packages/block-editor/src/components/link-control/index.js

```diff
@@ -530,10 +537,6 @@ function LinkControl( {
 		);
 	};
 
-	const viewerSlotFillProps = useMemo(
-		() => ( { url: value && value.url } ),
-		[ value && value.url ]
-	);
 	return (
```

Comment:
If directEntry allows you to enter a URL directly (?) then we should probably update this text too. "Search for Post" might be even better if we can pass a label through props

<img width="256" alt="Screen Shot 2020-05-22 at 1 15 59 PM" src="https://user-images.githubusercontent.com/6265975/82706513-a3472980-9c2e-11ea-82b9-279157cabfb8.png">

Reply:
Updated